templates/part.eventsrepeat.php

```diff
@@ -0,0 +1,635 @@
+</fieldset>
+
+
+
+
+<fieldset class="event-fieldset">
+	<label class="label"><?php p($l->t('Interval'))?></label>
+	<input type="number" min="1" max="1000" value="1" name="interval" ng-model="intervalmodel">
+</fieldset>
+
+
+
+<fieldset class="event-fieldset">
+	<label class="label"><?php p($l->t('End'))?></label>
+	<select>
+		<option ng-repeat="end in ender" value="end.val" ng-model="end.val">{{ end.displayname }}</option>
+	</select>
+</fieldset>
+<!--
+		<div id="advanced_options_repeating">
+			<table style="width:100%">
+				<tbody><tr id="advanced_month" >
+					<th width="75px"></th>
+					<td>
+						<select id="advanced_month_select" name="advanced_month_select">
+							<option value="monthday" selected="selected">by monthday</option>
+<option value="weekday">by weekday</option>
+						</select>
+					</td>
+				</tr>
+			</tbody></table>
+			<table style="width:100%">
+				<tbody><tr id="advanced_year" >
+					<th width="75px"></th>
+					<td>
+						<select id="advanced_year_select" name="advanced_year_select">
+							<option value="bydate" selected="selected">by events date</option>
+<option value="byyearday">by yearday(s)</option>
+<option value="byweekno">by weeknumber(s)</option>
+<option value="bydaymonth">by day and month</option>
+						</select>
+					</td>
+				</tr>
+			</tbody></table>
+			<table style="width:100%">
+				<tbody><tr id="advanced_weekofmonth" >
+					<th width="75px"></th>
+					<td id="weekofmonthcheckbox">
+						<select id="weekofmonthoptions" name="weekofmonthoptions">
+							<option value="auto" selected="selected">events week of month</option>
+<option value="1">first</option>
+<option value="2">second</option>
+<option value="3">third</option>
+<option value="4">fourth</option>
+<option value="5">fifth</option>
+<option value="-1">last</option>
+						</select>
+					</td>
+				</tr>
+			</tbody></table>
+			<table style="width:100%">
+				<tbody><tr id="advanced_weekday" >
+					<th width="75px"></th>
+					<td id="weeklycheckbox">
+						<select id="weeklyoptions" name="weeklyoptions[]" multiple="multiple" title="Select weekdays">
+							<option value="Monday">Monday</option>
+<option value="Tuesday" selected="selected">Tuesday</option>
+<option value="Wednesday">Wednesday</option>
+<option value="Thursday">Thursday</option>
+<option value="Friday">Friday</option>
+<option value="Saturday">Saturday</option>
+<option value="Sunday">Sunday</option>
+						</select><button type="button" class="ui-multiselect ui-widget ui-state-default ui-corner-all" title="Select weekdays" aria-haspopup="true" style="width: 156px;"><span class="ui-icon ui-icon-triangle-2-n-s"></span><span>Tuesday</span></button>
+					</td>
+				</tr>
+			</tbody></table>
+			<table style="width:100%">
+				<tbody><tr id="advanced_byyearday" >
+					<th width="75px"></th>
+					<td id="byyeardaycheckbox">
+						<select id="byyearday" name="byyearday[]" multiple="multiple" title="Select days">
+							<option value="1">1</option>
```

Comment:
Yes, this will go, every commented area is still not checked by me. Will squash commits later on.